fe/src/main/java/org/apache/doris/common/util/DynamicPartitionUtil.java

```diff
@@ -114,6 +118,34 @@ private static void checkEnable(String enable) throws DdlException {
         }
     }
 
+    private static void checkStartDayOfMonth(String val) throws DdlException {
+        if (Strings.isNullOrEmpty(val)) {
+            throw new DdlException("Invalid properties: " + DynamicPartitionProperty.START_DAY_OF_MONTH);
+        }
+        try {
+            int dayOfMonth = Integer.parseInt(val);
+            if (dayOfMonth < 1 || dayOfMonth > 28) {
```

Comment:
add a comment why less than 28, the reason in doc is for users?

Reply:
I have already stated the reason in the document, I will add some comments here.